pages/dev/gmp/debugging-failed-transaction.md

```diff
@@ -0,0 +1,47 @@
+# Debugging Failed Transaction
```

Comment:
two comments:
1. let's move this page inside of "Monitoring and Recovery"
2. change name of this page to "Debugging your Smart Contract"